### ISSUE TITLE
No value present error

### ISSUE THREAD

**Issue by tasomaniac** [RBusarow/ModuleCheck#399]
I tried the plugin in a project and getting this error. Unfortunately I cannot provide a sample project but I hope the stacktrace will be helpful. Please let me know if you would like to have more details. Thanks. 

```
Caused by: java.util.NoSuchElementException: No value present
        at modulecheck.parsing.java.JavaFile$parsed$2.invoke(JavaFile.kt:77)
        at modulecheck.parsing.java.JavaFile$parsed$2.invoke(JavaFile.kt:47)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at modulecheck.parsing.java.JavaFile.getParsed(JavaFile.kt:47)
        at modulecheck.parsing.java.JavaFile.access$getParsed(JavaFile.kt:34)
        at modulecheck.parsing.java.JavaFile$imports$2.invoke(JavaFile.kt:89)
        at modulecheck.parsing.java.JavaFile$imports$2.invoke(JavaFile.kt:88)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at modulecheck.parsing.java.JavaFile.getImports(JavaFile.kt:88)
        at modulecheck.api.context.Imports$Key.invoke(Imports.kt:43)
        at modulecheck.api.context.Imports$Key.invoke(Imports.kt:34)
        at modulecheck.parsing.ProjectContextImpl.get(ProjectContext.kt:40)
        at modulecheck.api.RealAndroidMcProject.get(RealAndroidMcProject.kt:42)
        at modulecheck.api.context.ImportsKt.getImports(Imports.kt:58)
        at modulecheck.api.context.ImportsKt.importsForSourceSetName(Imports.kt:60)
        at modulecheck.core.internal.UsesKt.usesInConfig(uses.kt:51)
        at modulecheck.core.internal.UsesKt.uses(uses.kt:35)
        at modulecheck.core.rule.InheritedDependencyRule.check(InheritedDependencyRule.kt:37)
        at modulecheck.gradle.task.ModuleCheckAllTask.evaluate(ModuleCheckAllTask.kt:37)
        at modulecheck.core.ModuleCheckRunner.run-IoAF18A(ModuleCheckRunner.kt:56)
        at modulecheck.gradle.task.ModuleCheckTask.run(ModuleCheckTask.kt:80)
```

**Comment by RBusarow** [comment]
Thanks for the report!

I was able to come up with a simple reproducer thanks to the stacktrace.  The fix is here: #400.

This is happening because your project has a `.java` file without a package declaration, and that case wasn't handled properly in my java parsing.

I've published a new snapshot with the fix.  If you would like to try version `0.11.4-SNAPSHOT`, this bug won't cause you any more problems.

I hope to be releasing 0.11.4 pretty soon.
